pyg_lib/csrc/random/cpu/biased_sampling.cpp

```diff
@@ -0,0 +1,60 @@
+#include "pyg_lib/csrc/random/cpu/biased_sampling.h"
+
+namespace pyg {
+namespace random {
+
+at::Tensor biased_to_cdf(at::Tensor rowptr, at::Tensor bias) {
+  TORCH_CHECK(rowptr.is_cpu(), "'rowptr' must be a CPU tensor");
+  TORCH_CHECK(bias.is_cpu(), "'bias' must be a CPU tensor");
+
+  auto cdf = at::empty_like(bias);
+  int64_t rowptr_size = rowptr.size(0);
+  int64_t* rowptr_data = rowptr.data_ptr<int64_t>();
```

Comment:
I would like to keep `rowptr` data-types dynamic. How about we wrap this inside a `AT_DISPATCH_INTEGRAL_TYPES` call?